TF_Settings_and_Tooling_Unity/Assets/TouchFree/SettingsUI/Scripts/DiagnosticAPI.cs

```diff
@@ -191,7 +195,33 @@ void HandleMessage(string _message)
                     Debug.Log("DiagnosticAPI - Could not parse device info response: " + _message);
                 }
                 break;
+            case "GetAllowImages":
+                try
+                {
+                    GetAllowImagesResponse data = JsonUtility.FromJson<GetAllowImagesResponse>(_message);
+                    allowImages = data?.payload ?? false;
+                    OnAllowImagesResponse?.Invoke();
+                }
+                catch (Exception ex)
+                {
+                    Debug.Log("DiagnosticAPI - Could not parse allow images response: " + _message);
+                }
+                break;
+            case "GetCameraOrientation":
+                try
+                {
+                    GetCameraOrientationResponse data = JsonUtility.FromJson<GetCameraOrientationResponse>(_message);
+                    cameraReversed = data?.payload.camera_orientation == "fixed-inverted";
+                    OnCameraOrientationResponse?.Invoke();
+                }
+                catch (Exception ex)
+                {
+                    Debug.Log("DiagnosticAPI - Could not parse camera orientation response: " + _message);
+                }
+                break;
             case "SetAnalyticsEnabled":
+            case "SetCameraOrientation":
+            case "SetAllowImages":
                 // No current use for this
```

Comment:
Does this mean we don't expect a response to these requests?

Reply:
I've put a more useful message in the comments for this. We are expecting a response for these requests but don't need to consume them